docs/services/web_power_switch.rst

```diff
@@ -1,14 +1,47 @@
 Web Power Switch
```

Comment:
Same here, can you confirm no driver installation is required, or alternately state here that it is required?